packages/python-packages/apiview-copilot/src/_apiview_reviewer.py

```diff
@@ -1,13 +1,18 @@
-import dotenv
+from azure.cosmos import CosmosClient
+from azure.search.documents import SearchClient
+from azure.search.documents.models import VectorizableTextQuery, QueryType, QueryCaptionType
```

Comment:
Do we need to add cosmos and search to the requirements.txt?

Reply:
Done